crates/subspace-networking/src/create.rs

```diff
@@ -340,7 +340,7 @@ where
             general_target_connections: SWARM_TARGET_CONNECTION_NUMBER,
             special_target_connections: SWARM_TARGET_CONNECTION_NUMBER,
             bootstrap_addresses: Vec::new(),
-            kademlia_mode_override: None,
+            kademlia_mode: Some(Mode::Server),
```

Comment:
Hm... I recall we wanted the default to be client, not server. Why this change? If we want to keep libp2p's default then we need to leave `None`.